scripts/check_deployed_market.py

```diff
@@ -0,0 +1,162 @@
+from brownie import accounts, Contract
+import numpy as np
+from pytest import approx
+
+OVL_ADDR = '0x4305C4Bc521B052F17d389c2Fe9d37caBeB70d54'
```

Comment:
The main comment is that this tests only a single deployed market, feed, with given risk parameters. So that in order to use it for different markets, we need to copy-paste the correct addresses and params in, every time. 

This is a good start though and can be easily extended

Reply:
Yeah agreed. Ideally there should be a separate file for params which is gitignored. Can create a loop for multiple markets as well